src/modules/commander/Commander.hpp

```diff
@@ -98,6 +98,17 @@ class Commander : public ModuleBase<Commander>, public ModuleParams
 private:
 
 	DEFINE_PARAMETERS(
+
+		(ParamInt<px4::params::NAV_DLL_ACT>) _datalink_loss_action,
```

Comment:
If this came to the commander param list, why not change the suffix to `COM`?

Reply:
They've already effectively been commander params for several releases. The only reason I didn't do the rename is that users will lost existing settings. This is the type of thing we can do for a release with proper migration documentation.